### ISSUE TITLE
Trying to run from git

### ISSUE THREAD

**Issue by robson-alcantara** [NearbyShops/Nearby-Shops-Android-app#24]
Hi guys,

I tried to compile and run the source code, but the system triggered the error below when trying to choose the location

E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.nearbyshops.enduserappnew, PID: 10718
    android.view.InflateException: Binary XML file line #162 in org.nearbyshops.enduserappnew:layout/fragment_place_picker_google: Binary XML file line #162 in org.nearbyshops.enduserappnew:layout/fragment_place_picker_google: Error inflating class fragment
    Caused by: android.view.InflateException: Binary XML file line #162 in org.nearbyshops.enduserappnew:layout/fragment_place_picker_google: Error inflating class fragment
    **Caused by: java.lang.RuntimeException: API key not found.  Check that <meta-data android:name="com.google.android.geo.API_KEY" android:value="your API key"/> is in the <application> element of AndroidManifest.xml**

**Comment by SumeetMoray** [comment]
Commercial use now requires a paid license https://nearbyshops.org/pricing.html

If you are trying to use this project for commercial purpose. Please be aware that we dont allow free commercial use !

**Comment by robson-alcantara** [comment]
I understand, I saw the product license. I'm trying to run on my phone. In a next step I want to use the structure of the city hall to make it available for the commerce. Due to COVID-19 we are not opening commerce.

**Comment by SumeetMoray** [comment]
You mean you agree to the fee right ?

**Comment by SumeetMoray** [comment]
Even for non-commercial use also payment is required !

**Comment by robson-alcantara** [comment]
If necessary, I will forward the information to the mayor. However I saw that there are some situations that are free. But I didn't read in detail

**Comment by SumeetMoray** [comment]
We changed that ... now payment is required for non-commercial use also. 

Without financial support there would be nobody to maintain this app. Would you use a non-maintained project ?

**Comment by SumeetMoray** [comment]
This is a very small fee compared to what other platforms charge !

**Comment by robson-alcantara** [comment]
I talk to the mayor. Inform me the values that I pass on. We have difficulties paying in any currency other than ours due to legislation

**Comment by SumeetMoray** [comment]
You can access your marketplace through nearby shops app. That is free of cost. You dont need any payment.

**Comment by robson-alcantara** [comment]
Okay, I will continue to study the technical and business part of the application

**Comment by SumeetMoray** [comment]
Payment is required for White-Label customization of app. But if you access your marketplace through our Nearby Shops app. Payment is not required !

**Comment by robson-alcantara** [comment]
Correct, that's what I saw. At the moment we have no interest in customizing the app, perhaps in the future, after the pandemic. The changes I made was to execute on my phone, since it is the only one I have. I managed to execute it and saw that the Brazilian Portuguese translation is partial.

**Comment by SumeetMoray** [comment]
Yes ! 
You can download our app from google play - https://play.google.com/store/apps/details?id=org.nearbyshops.enduserappnew

You will see multiple-markets --- we can create a free marketplace for you also !

**Comment by robson-alcantara** [comment]
As I understand it, to create a market we have the option of using our infrastructure. It is?

**Comment by SumeetMoray** [comment]
Yes you can self-host your own server ... and the data is safe with you !
We only link to your market. We dont save your data. 

And you can use white-label option in future also ... that option is always available with you !

**Comment by SumeetMoray** [comment]
If you ever want to use white-label app you can use it ... just pay the license fee and you can use !

**Comment by prineee** [comment]
Hello, I am interested in your app, I have physical shop as well as too many shopkeepers are there in my contact. I want to use it kindly guide me with your free mode.
Thank you.
